curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/60a3e3396c7b40068ad69970.md

```diff
@@ -16,25 +16,19 @@ Add a `link` element to link your `styles.css` file. Set the `href` to `./styles
 Your code should have a `link` element.
 
 ```js
-assert(/<link/.test(code))
+assert(/<(?:link|LINK)/.test(code))
 ```
 
-Your `href` attribute should have the value `./styles.css`.
+Your `link` element should have a `rel` attribute with the value `stylesheet`.
 
 ```js
-assert(/href\s*=\s*('|")(\.\/)?styles\.css\1/.test(code));
+assert.match(code, /<(?:link|LINK)[\s\S]*?rel=('|"|`)stylesheet\1/)
```

Comment:
Why not just use `.` instead of `[\s\S]`?

Reply:
I will be honest I was lazy. I copied Tom code, but you are right for consistency in the codes at least

Reply:
I will implement it

Reply:
> Why not use `.` instead of `[\s\S]`

I misunderstood so hard 🤣, sorry about that.

to answer your question, the file [ferris wheel step 1](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/6140c7e645d8e905819f1dd4.md) has the same regex expression, I wanted the people who see both, see the same regex. I am down to change both of the file to include `.` in the regex, while making step 1 case sensitive. 

what do you think?

Reply:
Nobody will remember the regex from a different exercise. I don't think there's a need to expand the scope of this PR either.

For the case thing, I don't know if there's any consensus, but to me it would be better to turn the whole regex case insensitive, since other parts could have mixed or upper case and still be valid. Plus, it leaves the expression more readable.

Reply:
while testing I had issues passing `REL` and `HREF` so I added `(?:ref|REF)` and same for `href`

Reply:
I meant `i`, you're needlessly obfuscating it. But maybe it's best to leave the case changing out of this PR altogether.

Reply:
> you're needlessly obfuscating it.

true, this happens because of my lacking. my first try was making DOM selector but it wasn't working and I couldn't debug it, and I meddle with regex, which I am not skilled at.

> But maybe it's best to leave the case changing out of this PR altogether.

sure, issue wasn't about case insensitive.